### ISSUE TITLE
IdsEditor: Accessibility WCAG Guideline: 3.3.2 Labels or Instructions Defect

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1317]
Describe the bug : Edit fields does not have semantic labels.

To Reproduce
1. Launch the URL: https://main.wc.design.infor.com/ids-editor/label-state.html
2. Navigate to 2nd and 3rd edit field using arrow keys and verify"
3. Edit fields does not have semantic labels. 

Expected behavior :Edit fields should have a programmatically discernible label.

User Impact:  User will not be able to identify the element.

Screenshots If applicable, add screenshots to help explain your problem.: https://www.screencast.com/t/FZ4QcbK9Rrc7

Dev Recommendation:  Use "aria-label" attribute to create an invisible text label for screen readers to read.

Platform
* Windows 10/Chrome/NVDA

Additional context
NA

**Comment by tmcconechy** [comment]
I think this can be fixed by actually adding a label value. However in the hidden example there is no label on purpose (not a common use case)
